spec/views/hyrax/admin/admin_sets/index.json.erb_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe "hyrax/admin/admin_sets/index.json.jbuilder" do
+  context "when no admin sets exists" do
+    it "renders no admin sets" do
+      render
```

Comment:
Just a style suggestion, but you could have put the `render` in a `before` block and the `json` in a `let` block. then only assertions are in the `it` block.